src/category_theory/triangulated/basic.lean

```diff
@@ -30,8 +31,9 @@ open category_theory.category
 We work in an additive category C equipped with an additive shift.
 -/
 variables (C : Type u) [category.{v} C] [has_shift C] [additive_category C]
-[functor.additive (shift C).functor]
+[functor.additive (shift C).functor] [functor.additive (shift C).inverse]
```

Comment:
I wonder if we could install `equivalence.additive` for precisely this situation.

Reply:
I had a go at proving that equivalences of additive categories are additive functors (using https://stacks.math.columbia.edu/tag/0DLP) but I am not familiar with the relevant parts of mathlib yet (mostly category_theory/limits/preserves), and I got stuck.